extensions/open-mysql-db/README.md

```diff
@@ -30,10 +30,12 @@ openmldb.password=root
 Use the following command to start the server.
 
 ```
-java -jar ./open-mysql-db-0.1.0-SNAPSHOT-jar-with-dependencies.jar
```

Comment:
benefit of the new approach?

Reply:
user can put properties file in any directory